src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -0,0 +1,53 @@
+import { incidentDate, secondaryIncidentDate } from '../../pages';
+
+import { isAnswering781Questions, isAnswering781aQuestions } from '../../utils';
+
+const numberToWords = {
+  0: 'First',
+  1: 'Second',
+  2: 'Third',
+  3: 'Fourth',
+  4: 'Fifth',
+  5: 'Sixth',
+  6: 'Seventh',
+  7: 'Eighth',
+  8: 'Ninth',
+  9: 'Tenth',
+};
+
+export function createFormConfig781(iterations) {
+  let configObj = {};
+  for (let index = 0; index < iterations; index++) {
+    configObj = {
+      ...configObj,
```

Comment:
So I think this works right now, but my concern is that the shallow copying of the existing object into the new one could lead to some weird results. I think in the context of a `for` loop, something like 
```
let configObj = {};
for (let index = 0; index < iterations; index++) {
    configObj[`incidentDate${index}`]: { /* Schema goes here! */ }
}
```

would be clearer and wouldn't risk something weird going on with the shallow copy in case the schema changes in the future.

Reply:
I'm not sure how this "solves" anything? Also the readability once there is more than one page becomes way more complicated. There is also the fact that we need to keep these pages in order which will become more difficult to maintain once we get more than one page.